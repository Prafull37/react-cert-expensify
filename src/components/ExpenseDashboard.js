import React from 'react';
import ExpenseList from './child/ExpenseList';
import ExpenseListFilters from "./child/ExpenseListFilter";

const ExpenseDashboard=(props)=>(<div>
    <ExpenseListFilters/>
    <ExpenseList/>
</div>)

export default ExpenseDashboard;