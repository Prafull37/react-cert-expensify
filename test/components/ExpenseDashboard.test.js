import React from "react"
import {shallow} from "enzyme";
import ExpenseDashboard from "../../src/components/ExpenseDashboard";

test('should render ExpenseDashboard correctly', () => {
    const wrapper = shallow(<ExpenseDashboard />);
    expect(wrapper).toMatchSnapshot();
  });