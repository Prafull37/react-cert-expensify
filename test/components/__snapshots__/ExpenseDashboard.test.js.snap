// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboard correctly 1`] = `
<div>
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
